corehq/apps/accounting/static/accounting/js/renew_plan_selection.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+hqDefine('accounting/js/renew_plan_selection', [
+    'jquery',
+    'knockout',
+    'hqwebapp/js/initial_page_data',
+    'hqwebapp/js/toggles',
+], function (
+    $,
+    ko,
+    initialPageData,
+    toggles
+) {
+    const PlanRenewalView = function (options) {
+        const self = this;
+        self.monthlyPlan = options.renewalChoices.monthly_plan;
+        self.annualPlan = options.renewalChoices.annual_plan;
+        self.showAnnualPlan = ko.observable(options.isAnnualPlan);
+        self.selectedPlan = ko.pureComputed(() => {
```

Comment:
@nospame I think requirejs will be very upset about about the es6 spread operator. Have you deployed this code to staging? I'm pretty sure the requirejs build will break

Reply:
this is why we need webpack lol

Reply:
Yes, this is on staging now with no apparent issues in deploy or use, but happy to move to pre-es6 syntax here.

Reply:
If it deployed successfully to staging it's fine to leave it. We use arrow functions in other places.